### PR TITLE
Fix virtual cluster redeploy script

### PR DESCRIPTION
**Pull request by SchSeba**

### CODE REVIEW

File: hack/virtual-cluster-redeploy.sh [hunk 1]
```diff
@@ -19,10 +19,14 @@ if [ $CLUSTER_TYPE == "openshift" ]; then
   echo ${auth} > registry-login.conf
 
   internal_registry="image-registry.openshift-image-registry.svc:5000"
-  pass=$( jq .\"$internal_registry\".password registry-login.conf )
+  pass=$( jq .\"image-registry.openshift-image-registry.svc:5000\".auth registry-login.conf  )
```
Comment (adrianchiris) [inline hack/virtual-cluster-redeploy.sh:22]: you can use jq to slice the string value e.g ` jq '.[1:-1]'` then pipe to base64. just suggestion current way also works.

File: hack/virtual-cluster-redeploy.sh [hunk 2]
```diff
@@ -67,9 +72,38 @@ if [ $CLUSTER_TYPE == "openshift" ]; then
   export SRIOV_NETWORK_OPERATOR_IMAGE="image-registry.openshift-image-registry.svc:5000/$NAMESPACE/sriov-network-operator:latest"
   export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE="image-registry.openshift-image-registry.svc:5000/$NAMESPACE/sriov-network-config-daemon:latest"
   export SRIOV_NETWORK_WEBHOOK_IMAGE="image-registry.openshift-image-registry.svc:5000/$NAMESPACE/sriov-network-operator-webhook:latest"
-fi
+  echo "## deploying SRIOV Network Operator"
+  hack/deploy-setup.sh $NAMESPACE
+else
+  source hack/env.sh
```
Comment (adrianchiris) [inline hack/virtual-cluster-redeploy.sh:78]: should we have hack/deploy-operator-helm.sh with this logic ? then use it here and in run-e2e-conformance-virtual ?

Comment (SchSeba) [reply hack/virtual-cluster-redeploy.sh]: nice idea! done

File: hack/virtual-cluster-redeploy.sh (hunk 1 above)
Comment (SchSeba) [reply hack/virtual-cluster-redeploy.sh:22]: I will leave this one for now if that is ok :)

File: hack/deploy-operator-helm.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  --set images.webhook=${SRIOV_NETWORK_WEBHOOK_IMAGE} \
+  --set operator.admissionControllers.enabled=${ADMISSION_CONTROLLERS_ENABLED} \
+  --set operator.admissionControllers.certificates.certManager.enabled=${ADMISSION_CONTROLLERS_CERTIFICATES_CERT_MANAGER_ENABLED} \
+  --set sriovOperatorConfig.deploy=true"
+
+PATH=$PATH:${root}/bin
+make helm
+helm  upgrade -n ${NAMESPACE} --create-namespace \
+  $HELM_VALUES_OPTS \
+  --wait sriov-network-operator ./deployment/sriov-network-operator-chart
```
Comment (adrianchiris) [inline hack/deploy-operator-helm.sh:36]: nit: missing newline at the end.

**Review by adrianchiris — COMMENTED** [review]
minor nit otherwise LGTM

File: hack/deploy-operator-helm.sh (hunk 3 above)
Comment (SchSeba) [reply hack/deploy-operator-helm.sh]: yep :)

**Review by adrianchiris — APPROVED** [review]

**Review by zeeke — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**